vuepress/docs/next/tutorials/curate/local-hub-private-git-repo.md

```diff
@@ -2,11 +2,11 @@
 # Install Bundles from a Private Git Repository
 
 ## Overview
-The standard deployment of the Entando Component Repository (ECR) assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication, but if a customer chooses to limit repository access to specific Git accounts, a private key is required. This document details how to allow the ECR to access a private Git repository.
+The standard deployment of the Local Hub assumes that bundles are checked out from public Git repositories. Public repositories do not require user authentication, but if a customer chooses to limit repository access to specific Git accounts, a private key is required. This document details how to allow the Local Hub to access a private Git repository.
```

Comment:
Add a TODO to refine this language for git vs docker

Reply:
it looks like the whole tutorial needs to be reworked to include instructions for docker...

Reply:
We don't need to rewrite this tutorial since it is specifically for git-based bundles. I just wanted to refine the language to note that in Entando 7.1 you can also do this with docker.